src/views/Voting_old.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div class="proposals view-main">
+    <div class="fx-row mg-b32">
+      <h2 class="view-title">Voting</h2>
+    </div>
+    <div class="app-table">
+      <div class="proposals__table-head app-table__head">
+        <div class="app-table__cell">
+          <span class="app-table__cell-txt"> ID </span>
+        </div>
+        <div class="app-table__cell">
+          <span class="app-table__cell-txt"> Title </span>
+        </div>
+        <div class="app-table__cell">
+          <span class="app-table__cell-txt"> Tally </span>
+        </div>
+        <div class="app-table__cell">
+          <span class="app-table__cell-txt"> Deposit </span>
+        </div>
+        <div class="app-table__cell">
+          <span class="app-table__cell-txt"> Status </span>
+        </div>
+      </div>
+      <template v-if="proposals?.length">
+        <button
+          v-for="(item, idx) in proposals"
+          :key="item.id"
+          class="app-table__row-btn"
+          type="button"
+          @click="showProposal(item, tallies[idx])"
+        >
+          <div class="proposals__table-row app-table__row">
+            <div class="app-table__cell">
+              <TitledSpan
+                class="app-table__cell-txt"
+                :text="item.proposalId.toString()"
```

Comment:
better to set titledSpan component accept string and number props
:text={ type: [String, Number], ... }